data/filetype_extensions.conf

```diff
@@ -47,7 +47,7 @@ JSON=*.json;
 Julia=*.jl;
 Kotlin=*.kt;*.kts;
 LaTeX=*.tex;*.sty;*.idx;*.ltx;*.latex;*.aux;
-Lisp=*.lisp;
+Lisp=*.lisp;*.el
```

Comment:
ending semicolon?

Reply:
I am not familiar with GitHub (web) interface.
As I see in the "Commits" section the semicolon is no added, right?
But I can not see it here in the "Conversations".

Is it OK that way?

Reply:
> But I can not see it here in the "Conversations".

Note the "outdated" above :smile:  

The "Files Changed" tab shows the aggregate of all commits and is fine.

Note the semicolon is not required, but every other line has one, so better to be consistent to avoid questions about why.

Reply:
> Because of that and if you agree I would create a follow-up PR removing all line-ending semicolon.

I suggest saving you and us the effort to create and discuss such a PR. IMO it is just not important if there are terminating line ending semi colons or not. Since they are there now, I'd say just keep it as it is and as it is also for probably most user configurations.